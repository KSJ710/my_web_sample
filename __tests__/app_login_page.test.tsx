import { render, screen } from '@testing-library/react'
import Page from 'app/login/page'

describe('login', () => {
  it('render page unchanged', () => {
    const { container } = render(<Page />)
    expect(container).toMatchSnapshot()
  })
})
