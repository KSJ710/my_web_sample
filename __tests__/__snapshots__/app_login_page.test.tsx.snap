// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login render page unchanged 1`] = `
<div>
  <form
    action=""
    class="m-auto max-w-[368px] px-6"
  >
    <label
      class="font_yellow_underline_green text-2xl"
      for="name"
    >
      ユーザー名
    </label>
    <input
      class="mb-[24px] block w-full border-[1px] text-2xl "
      name="name"
      type="text"
    />
    <label
      class="font_yellow_underline_green text-2xl"
      for="password"
    >
      パスワード
    </label>
    <input
      class="mb-[24px] block w-full border-[1px] text-2xl"
      name="password"
      type="text"
    />
    <a
      href="/adm"
    >
      <input
        class="bg-main p-[8px] text-lg text-accent"
        type="submit"
        value="ログインする"
      />
    </a>
  </form>
</div>
`;
